Makefile

```diff
@@ -2,23 +2,20 @@ current_user  := $(shell id -u)
 current_group := $(shell id -g)
 BUILD_DIR     := $(PWD)
 DOCKER_FLAGS  := --interactive --tty
-DOCKER_IMAGE  := wikimediade/fundraising-frontend
+DOCKER_IMAGE  := registry.gitlab.com/fun-tech/fundraising-frontend-docker
 
 install-php:
 	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer install $(COMPOSER_FLAGS)
 
 update-php:
 	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer update $(COMPOSER_FLAGS)
 
-ci: covers phpunit cs stan
+ci: phpunit cs stan
 
-test: covers phpunit
-
-covers:
-	docker-compose run --rm app ./vendor/bin/covers-validator
+test: phpunit
 
 phpunit:
-	docker-compose run --rm app ./vendor/bin/phpunit
+	docker-compose run --rm -e XDEBUG_MODE=coverage app ./vendor/bin/phpunit
```

Comment:
For regular PHPUnit runs (in images without XDebug and phpunit calls without coverage), this parameter does nothing (except maybe future-proof things or confuse developers, depending on how you look at it).

This repos travis file uses "native" PHP instead of running it in a container